index.bs

```diff
@@ -2387,6 +2442,84 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Deserialize Registration ceremony options - PublicKeyCredential's `getCreationOptionsFromJSON()` Method ### {#sctn-getCreationOptionsFromJSON}
```

Comment:
IMHO, `PublicKeyCredential.publicKeyCredentialCreationOptionsFromJSON()` would be the right choice here. (And `publicKeyCredentialRequestOptionsFromJSON` for the assertion case.) I saw the comments on the main thread, and I agree it's slightly wordy. But if nothing else, referencing the exact name makes it more obvious what this method is supposed to do, and it aids grep-ability. And most web app code bases will call this method at most once, so the benefit of a shorter name isn't that huge. :-)